### PR TITLE
Monitor test case refactor 588

### PR DESCRIPTION
**Pull request by pras7430**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-588

 ## Description:
1. Test cases update and as per refactoring of monitor and resource service.
2. Helix version updated.
3. Merged Server-error.intercpetor in request.interceptor.

 ## Testing:
npm run test:unit
 ## Screenshots:
(if applicable)

### CODE REVIEW

File: src/app/_features/monitors/components/list/monitorslist.component.spec.ts [hunk 1]
```diff
@@ -148,45 +158,40 @@ describe('MonitorslistComponent', () => {
...
+    expect(spy).toHaveBeenCalledTimes(component.selectedMonitors.length);
   });
 
   it('should execute progress bar for success', () => {
     let obj = {id:[{id: "889EJ382", name: "Bandwidth Monitoring for eth0", labelSelectorMethod: "AND", interval: "30", labelSelector: {additionalProp1: "UbuntuOS", additionalProp2: "Prod", additionalProp3: "DockerApps"}, createdTimestamp: "2019-12-31T19:04:51Z", updatedTimestamp: "2020-01-03T18:50:16Z"}], error: true};       
     let count = 0;
     count++;
     component.progressBar(count, obj);
-    expect(component.selectedMonForDeletion).toEqual([obj]);    
+    console.log(component.selectedMonForDeletion);
```
Comment (lovedeep92) [inline src/app/_features/monitors/components/list/monitorslist.component.spec.ts:193]: Please remove this console from here!

File: src/app/_features/monitors/components/monitor-search/monitor-search.component.spec.ts [hunk 2]
```diff
@@ -24,16 +24,21 @@ describe('MonitorSearchComponent', () => {
...
-
-  beforeEach(() => {
-
     injector = getTestBed();
     monitorsService = injector.get(MonitorService);
     fixture = TestBed.createComponent(MonitorSearchComponent);
     component = fixture.componentInstance;
     fixture.detectChanges();
-  });
+  }));
```
Comment (lovedeep92) [inline src/app/_features/monitors/components/monitor-search/monitor-search.component.spec.ts:32]: I think we can remove this commented code as it is not required now?

File: src/app/_features/monitors/components/select/select.component.spec.ts [hunk 3]
```diff
@@ -34,7 +32,18 @@ describe('SelectComponent', () => {
       placeholder: '',
       inputType: ''
     });
-  });
+  }));
+
+  // beforeEach(() => {
```
Comment (lovedeep92) [inline src/app/_features/monitors/components/select/select.component.spec.ts:37]: Same here!

File: src/app/_features/monitors/pages/details/monitor-details.page.spec.ts [hunk 4]
```diff
@@ -144,7 +144,21 @@ describe('MonitorDetailComponent', () => {
     });
     fixture.detectChanges();
     component.monDetails = new monitorsMock().single;
-  });
+  }));
+
+  // beforeEach(() => {
```
Comment (lovedeep92) [inline src/app/_features/monitors/pages/details/monitor-details.page.spec.ts:149]: same here!

File: src/app/_features/monitors/pages/monitors/monitors.page.spec.ts [hunk 5]
```diff
@@ -9,16 +10,20 @@ describe('MonitorsPage', () => {
...
-  }));
-
-  beforeEach(() => {
     fixture = TestBed.createComponent(MonitorsPage);
     component = fixture.componentInstance;
     fixture.detectChanges();
-  });
+  }));
+
+  // beforeEach(() => {
```
Comment (lovedeep92) [inline src/app/_features/monitors/pages/monitors/monitors.page.spec.ts:22]: here too!

File: src/app/_features/resources/components/list/resourceslist.component.spec.ts [hunk 6]
```diff
@@ -85,7 +83,23 @@ describe('ResourcesListComponent', () => {
     });
 
     fixture.detectChanges();
-  });
+  }));
+
+  // beforeEach(() => {
```
Comment (lovedeep92) [inline src/app/_features/resources/components/list/resourceslist.component.spec.ts:88]: +1

**Review by FarajiA — APPROVED** [review]
Wow! That was a big one. Great **JOB**
wheww

**Pull request merged**